packages/core/core/src/useTablePreferences/types/index.ts

```diff
@@ -5,4 +5,8 @@ export interface UserTablePreferences {
   sortColumnKey?: string
   /** The order by which to sort the column, one of `asc` or `desc` */
   sortColumnOrder?: 'asc' | 'desc'
+  /** The customized column widths, if resizing is allowed */
+  columnWidths?: Record<string, number>
```

Comment:
suggestion: delete this definition and use the type export directly from kongponents.

https://github.com/Kong/konnect-ui-apps/blob/main/apps/gateway-manager/src/components/data-plane-nodes/RuntimeInstances.vue#L550

Reply:
Done

Reply:
Deleted, and using type directly from kongponents